components/v2/GameThumbnail/GameThumbnail.tsx

```diff
@@ -2,42 +2,40 @@
 import { GameThumbnailPropsI } from './GameThumbnailProps';
 
 import styles from './GameThumbnail.module.scss';
-import Link from 'next/link';
 import { useAccount } from 'wagmi';
 import { useEffect, useState } from 'react';
 import { getRulesContract } from 'gameApi';
 import { TGameType } from 'types/game';
 import { useTranslation } from 'react-i18next';
 import { SelectPrize } from '../SelectPrize';
 import Blockies from 'react-blockies';
-import {shortenAddress} from 'helpers/utils';
+import { shortenAddress } from 'helpers/utils';
+import Link from 'next/link';
 
 export const GameThumbnail: React.FC<GameThumbnailPropsI> = ({
   name,
   image,
   url,
   description,
-  isTransactionPending,
   setIsTransactionPending,
-  isRequestConfirmed,
   setIsRequestConfirmed,
-  transactionLink,
   setTransactionLink,
+  openWalletModal,
 }) => {
   const { t } = useTranslation();
   const [rulesAddress, setRulesAddress] = useState('');
   const [isStartButtonOpen, setIsStartButtonOpen] = useState(false);
 
   const gameName = t(`gameTypePage.games.${name}`);
-  const game = url as TGameType;
+  const gameType = url as TGameType;
```

Comment:
consider naming: TGameType is a kind of one-word-string, and url is a url. Maybe find an bit more appropriate name for the "url" prop?

Reply:
could be gameUrl or linkGame?